betterlint.gemspec

```
@@ -3,7 +3,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |s|
   s.name = "betterlint"
-  s.version = "1.2.2"
+  s.version = "1.2.3"
```

```suggestion
  s.version = "1.3.0"
```

Incrementing minor version here is more inline with how rubocop adds new rules in minor releases.